packages/unmock-node/src/__tests__/backend/index.test.ts

```diff
@@ -8,19 +8,23 @@ const servicesDirectory = path.join(__dirname, "..", "loaders", "resources");
 describe("Node.js interceptor", () => {
   describe("with petstore in place", () => {
     let nodeInterceptor: NodeBackend;
+
     beforeAll(() => {
       nodeInterceptor = new NodeBackend({ servicesDirectory });
       const unmockOptions = new UnmockOptions();
       nodeInterceptor.initialize(unmockOptions);
     });
+
     afterAll(() => {
       nodeInterceptor.reset();
     });
+
     test("gets successful response for valid request", async () => {
       const response = await axios("http://petstore.swagger.io/v1/pets");
       expect(response.status).toBe(200);
       expect(response.data).toBeDefined();
     });
+
```

Comment:
Could also add a few asserts for `response.data` to be of the correct format (if the generator now knows how to create proper data).

Reply:
:+1: tests are not finished just yet :)